frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditOptions.module.css

```
@@ -1,11 +1,24 @@
 .root {
-  margin-top: var(--fds-spacing-5);
-  border-top: 1px solid var(--fds-semantic-border-divider-default);
-  padding: var(--fds-spacing-5) 0 0;
+  margin-top: var(--fds-spacing-2);
+  padding: var(--fds-spacing-2) 0 0;
+  background-color: white;
+  border-radius: var(--fds-border_radius-medium);
+  box-shadow: var(--fds-shadow-small);
+  margin: var(--studio-property-button-vertical-spacing) var(--fds-spacing-5);
+  padding: var(--data-model-binding-spacing);
+  gap: var(--data-model-binding-spacing);
```

`--data-model-binding-spacing` does not seem to exist in this context

```suggestion
  padding: var(--fds-spacing-3);
  gap: var(--fds-spacing-3);
```